synapse/events/third_party_rules.py

```diff
@@ -38,7 +40,7 @@ def __init__(self, hs):
 
         if module is not None:
             self.third_party_rules = module(
-                config=config, http_client=hs.get_simple_http_client()
+                config=config, module_api=ModuleApi(hs, hs.get_auth_handler()),
```

Comment:
I think you'll need to add a way to get hold of an http client to `ModuleApi` ?

Reply:
I've added `http_client` as a member variable to `ModuleApi`, but perhaps that's not very discoverable for users? Should I add a `get_http_client()` method or some such?